src/pages/Cart.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+import CartProduct from "../components/CartProduct";
+import "../style/cart.css";
+import { useCart } from "../context/cart-context";
+import PriceDetails from "../components/PriceDetails";
 
 function Cart() {
+  const {
+    state:{cart}
+  } = useCart();
   return (
-    <div>Cart</div>
-  )
+    <>
+      <h1>Cart</h1>
+
+      <div className="cart_container">
+        <div className="cart_main_container">
+
+          {cart.map((cartItem) => (            
+            <CartProduct
+              name={cartItem.name}
+              img={cartItem.img}
+              data={cartItem}
+              price={cartItem.price}
+              type={cartItem.type}
+              key={cartItem._id}
+              _id={cartItem._id}
+              qty={cartItem.quantity}
```

Comment:
Just send the data object and destructure it in the props

Reply:
thanks for the suggestion